### DISCUSSION TITLE
Mailgun alternative?

### DISCUSSION THREAD

**Discussion by caygri** [getfider/fider#1108] [General]
Hello, 

is possibile configure another email provider alternative to mailgun?

**Comment by goenning** [comment]
Only Mailgun, AWS SES and SMTP are supported.

**Comment by caygri** [comment]
thank you for your reply. where i found correct configuration of aws ses? I'm using pikapods. I create ses account but i dont receive email activation.

**Comment by boydthomson** [comment]
I'm using the SMTP settings with SendGrid.  You may have to disable the URL click tracking in SendGrid (https://www.twilio.com/docs/sendgrid/ui/account-and-settings/tracking#click-tracking)

**Reply by fadisawa** [reply]
Hi @boydthomson, I'm trying to use SMTP with Send Grid. what do you include in EMAIL_NOREPLY field?

**Reply by mattwoberts** [reply]
@fadisawa That's your default reply email address - something like noreply@yourcompany.com

**Comment by harrisjacktech-svg** [comment]
Yes, usually you can configure other email providers instead of Mailgun. A lot of people use SMTPmart, Mailjet, Postmark, SendGrid, or Resend depending on their needs.
